### DISCUSSION TITLE
Order of transformation parameters

### DISCUSSION THREAD

**Discussion by Misham2941** [SuperElastix/elastix#968] [Q&A] [answered]
Hi, I preformed rigid registration using elastix and I have been reviewing the transform parameters.
I came across these two lines:
(Transform "EulerTransform")
(TransformParameters 0.000314412349914827 0.00021504823325341053 -0.09517002733191654 -10.860687019455854 18.900506611419427 0.7732054479875283)

My question which set of values represent translation and rotation. Do the first 3 values represent translation and the last 3 represent rotation or is it the other way around? 

Thanks,
[TransformParameters.0.txt](https://github.com/SuperElastix/elastix/files/12746619/TransformParameters.0.txt)

**Comment by N-Dekker** [comment, marked as answer]
Thanks for asking, @Misham2941 Does the following piece of code answer your question?

https://github.com/SuperElastix/elastix/blob/c96f0266a0a10c274030f79ede8b88f9f2450a7c/Common/Transforms/itkAdvancedEuler3DTransform.hxx#L86-L91 
```

 this->m_Parameters[0] = m_AngleX; 
 this->m_Parameters[1] = m_AngleY; 
 this->m_Parameters[2] = m_AngleZ; 
 this->m_Parameters[3] = this->GetTranslation()[0]; 
 this->m_Parameters[4] = this->GetTranslation()[1]; 
 this->m_Parameters[5] = this->GetTranslation()[2];
```

Kind regards, Niels

**Reply by Misham2941** [reply]
Thanks N-Dekker, I am new to Elastix and image registration. When is registration preformed the transform parameters output txt is telling my how much it moved for registration?
